### ISSUE TITLE
[BUG] First load can't load photo from google place image because of the google restriction

### ISSUE THREAD

**Issue by vitvara** [ThaiRepose/thairepose#99]
first load
<img width="898" alt="image" src="https://user-images.githubusercontent.com/69972884/143474272-27059db7-087d-472d-a903-f927e30e3a04.png">
refresh
<img width="896" alt="image" src="https://user-images.githubusercontent.com/69972884/143474309-3983dfff-5bb6-449d-ae61-63046fd48c75.png">

**Comment by tboonma** [comment]
OK, I got it.
I've searched and found that The place autocomplete is a part of Google Maps JavaScript API, so an API key with HTTP referrer restriction works well.

The Google place photo is a part of Places API web service. The web services support only IP address restrictions, they will fail with HTTP referrer restrictions on API keys.

Reference: [from Stackoverflow](https://stackoverflow.com/questions/47754253/google-place-photo-returns-403-when-restricting-access-by-referer)
